crypto/bls/command/action_test.go

```diff
@@ -0,0 +1,151 @@
+package command
```

Comment:
In general, each single test function here is testing A LOT of different cases.

It'd be best to split out the test cases in different functions.
That way when a function fails, we immediately know the case that's affected.